docs/reference/mapping.asciidoc

```diff
@@ -68,6 +68,14 @@ via the <<multi-fields>> parameter.
 [float]
 === Settings to prevent mappings explosion
 
+Defining too many fields in an index or having too many nested documents or too many fields that contain nested
```

Comment:
I think we should only mention too many fields in an index here. how do too many nested documents cause a mapping explosion?